### PR TITLE
bpo-14265: Adds fully qualified test name to unittest output

### PR DESCRIPTION
**Pull request by dignissimus**
Amends unittest output to include the fully qualified test name

https://bugs.python.org/issue14265

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@dignissimus

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Review by MaxwellDupre — APPROVED** [review]
Add co author palaviv to ACKS.
Ran 55 tests in 0.033s
OK

**Comment by dignissimus** [comment]
Thank you for the review.

`palaviv` is already listed under ACKS as Aviv Palivoda.

File: Doc/library/unittest.rst [hunk 1]
```diff
@@ -139,12 +139,12 @@ line, the above script produces an output that looks like this::
...
-   test_isupper (__main__.TestStringMethods) ... ok
-   test_split (__main__.TestStringMethods) ... ok
-   test_upper (__main__.TestStringMethods) ... ok
+   test_isupper (__main__.TestStringMethods.test_isupper) ... ok
+   test_split (__main__.TestStringMethods.test_split) ... ok
+   test_upper (__main__.TestStringMethods.test_upper) ... ok
 
    ----------------------------------------------------------------------
-   Ran 3 tests in 0.001s
+   Ran 3 tests in 0.000s
```
Comment (asvetlov) [inline Doc/library/unittest.rst]:
```suggestion
   Ran 3 tests in 0.001s
```
Leave a small execution time even if your machine is super-fast.
It removes unnecessary user surprising.

**Comment by dignissimus** [comment]
Does anybody know what happened with the Windows x86 tests?

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**